ui-tests/src/test/java/org/openmrs/reference/FindPatientRecordTest.java

```diff
@@ -33,11 +34,12 @@ public void deletePatient() throws Exception {
         deletePatient(patient);
     }
 
-    @Ignore
-    @Category(org.openmrs.reference.groups.BuildTests.class)
-    public void testFindPatientRecord() throws InterruptedException {
+    @Test
+    @Category(BuildTests.class)
+    public void findPatientRecordTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
```

Comment:
Waiting the page to load ensures that the patient identifier entered matches with the test created patient identifier. Otherwise without waiting the page to load, the comparison is made against the entered patient identifier with any of the existing patient instead of the test created patient